lib/collection/src/operations/universal_query/planned_query.rs

```diff
@@ -55,7 +58,7 @@ pub enum Source {
     ScrollsIdx(usize),
 
     /// A nested prefetch
-    Prefetch(MergePlan),
+    Prefetch(Box<MergePlan>),
```

Comment:
Interesting clippy warning!

```
 1  warning: large size difference between variants
   --> lib/collection/src/operations/universal_query/planned_query.rs:53:1
    |
 53 | / pub enum Source {
 54 | |     /// A reference offset into the main search batch
 55 | |     SearchesIdx(usize),
    | |     ------------------ the second-largest variant contains at least 8 bytes
 ...  |
 61 | |     Prefetch(MergePlan),
    | |     ------------------- the largest variant contains at least 240 bytes
 62 | | }
    | |_^ the entire enum is at least 240 bytes
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.htm
 l#large_enum_variant
    = note: `#[warn(clippy::large_enum_variant)]` on by default
 help: consider boxing the large fields to reduce the total size of the enum
    |
 61 |     Prefetch(Box<MergePlan>),
    |              ~~~~~~~~~~~~~~
```

Based on that I do agree with the change. Not to make `SearchesIdx` and `ScrollsIdx` very heavy.

Reply:
My reasoning was that MergePlan only exists for queries with prefetch, so they will be heavy anyways.